website/api/api.md

```diff
@@ -0,0 +1,158 @@
+# API description
+For simplicity, answers can contain only this status codes:
+* `200 OK` if success;
+* `400 Bad Request` if there is request syntax error;
+* `500 Internal Server Error` if server error.
+
+## GET
+`GET api/quest/{quest_id}`
+
+Return JSON with information about quest.
+
+JSON contains fields as same as table [quests](../../docs/image/db.png),
+but with 
+field `start_qustion_id` and array `tags`, array 
+and also arrays `questions`, `answer_options`, `hints`, 
+`files`, `places`, `movements` with info about related
+entities
+(**but with values instead of values ids, e.g. `q_type` 
+instead of `q_type_id`**).
+
+`question`, `hint`, `answer` objects also contain `file_ids`
+key with ids of related files, described in array `files`.
+
+`question` objects contain arrays `hints_ids`, 
+`answer_option_ids`, `movements_ids`
+
+`movement` objects contain array `places_ids`.
+
+Returned JSON example:
+```json
+{
+  "title": "Example",
+  "author": "some author",
+  "description": "This is example quest",
+  "tags": ["example", "test"],
+  "password": "12345abc",
+  "hidden": true,
+  "file_ids": [1],
+  "time_open": "2011-01-01 00:00:00",
+  "time_close": "2022-01-01 00:00:00",
+  "lead_time": "6 months",
+  "start_question_id": 0,
+  "questions": [
+    {
+      "question_id": 0,
+      "question_text": "",
+      "q_type": "start",
```

Comment:
But what is type of qustion there? This is "open", "choice" or "movement"?

Reply:
Type "start". It is special fictive type. See db description.